packages/paste-website/src/component-examples/DetailTextExamples.ts

```diff
@@ -0,0 +1,158 @@
+export const detailTextWithImage = `
+const DetailTextExample = () => {
+  return (
+    <Box maxWidth='size30'>
+      <Box as='img' alt='' src='/images/detail-text/detail-text-image.png' maxWidth='100%' />
+      <DetailText>
+        Your sender identity is the “from” email address your recipients will see in their inbox.
+      </DetailText>
+    </Box>
+  );
+};
+render(<DetailTextExample />)
+`.trim();
+
+export const detailTextWithPrimaryContent = `
+const DetailTextExample = () => {
```

Comment:
todo: Detail Text in the Table example should have marginTop set to 0 https://www.figma.com/file/terVMAIFX264TQikusaqEr/detail-text?node-id=1%3A15204&t=ecKvbHJv2Yk64xpA-4